app/controllers/api/v1/recipes_controller.rb

```diff
@@ -4,9 +4,9 @@ def index
   end
 
   def show
-    render json: EdamamFacade.recipe_show(params[:ext_id])
+    render json: EdamamFacade.recipe_show(params[:id])
```

Comment:
The ext_id was in here because the edemam ids are strings and wont match up with our current recipe ids. If you pull just the id, this request will probably include a group in the params id, so something to look out for. The ext_id is what recipe_show uses to pull up the correct recipe from edamam